src/components/Image/Image.tsx

```diff
@@ -20,7 +20,7 @@ const Image = (props: ImageProps) => {
     const projectSettings = useContext(ProjectSettingsContext);
     const {
         src,
-        alt,
+        alt = 'image alt',
```

Comment:
Maybe we have to name alt consistent? I saw names started with Capital letters  and this one starts with lowercase letter

Reply:
done, Capital letter